packages/material-ui/src/Accordion/Accordion.js

```diff
@@ -77,7 +76,6 @@ const AccordionRoot = experimentalStyled(
           display: 'none',
         },
       },
-      /* Styles applied to the root element if `expanded={true}`. */
```

Comment:
+1 for removing these comments, I believe they are obsolete as the condition is right below them

Reply:
👍 , it's pretty much self-explanatory now with the `styleProps`. Also, it's no longer used to generate the documentation.